Libraries/LibWeb/HTML/Storage.h

```diff
@@ -49,6 +49,8 @@ class Storage : public Bindings::PlatformObject {
     void broadcast(StringView key, StringView old_value, StringView new_value);
 
     OrderedHashMap<String, String> m_map;
+    u64 m_quota_limit { 0 };
```

Comment:
I think `m_quota_bytes` would make more sense. "quota" already implies there's a limit and it wasn't immediately clear what unit we're working with.